### PR TITLE
Add Translate extension-related custom configuration for snapwikiwiki

### PR DESCRIPTION
**Pull request by R4356th**
Adds Translate extension-related custom configuration for snapwikiwiki where I am a bureaucrat. Documentation for these can be found in the [extension's extension.json](https://github.com/wikimedia/mediawiki-extensions-Translate/blob/master/extension.json#L858).

### CODE REVIEW

File: LocalWiki.php [hunk 1]
```diff
@@ -182,6 +182,14 @@ function wfDonateLink( $skin, &$bar ) {
 	unset( $wgGroupPermissions['checkuser'] );
 }
 
+if ( $wgDBname === 'snapwikiwiki' ) {
+	$wgTranslateDocumentationLanguageCode = 'qqq';
```
Comment (Universal-Omega) [inline LocalWiki.php:186]: This can be done via Special: ManageWiki. In addition, this value ("qqq") is the default.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalWiki.php (hunk 1 above)
Comment (R4356th) [reply LocalWiki.php:186]: qqq is not default.

Comment (Universal-Omega) [reply LocalWiki.php:186]: I noticed my mistake.

Comment (R4356th) [reply LocalWiki.php:186]: And apologies for not noticing this before.

Comment (Universal-Omega) [reply LocalWiki.php:186]: It should still be removed though since can be done in ManageWiki

Comment (R4356th) [reply LocalWiki.php]: Done.

File: LocalWiki.php [hunk 2]
```diff
@@ -182,6 +182,14 @@ function wfDonateLink( $skin, &$bar ) {
 	unset( $wgGroupPermissions['checkuser'] );
 }
 
+if ( $wgDBname === 'snapwikiwiki' ) {
```
Comment (Universal-Omega) [inline LocalWiki.php:185]: I'm pretty sure that these can rather be added to LocalSettings.php rather then here. Thanks!

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

File: LocalWiki.php (hunk 2 above)
Comment (R4356th) [reply LocalWiki.php:185]: You will have to wait for a few hours then.

**Comment by R4356th** [comment]
@Universal-Omega, would not adding this to ManageWiki be a better idea if we are actually moving this to LS.php?

**Comment by Universal-Omega** [comment]
> @Universal-Omega, would not adding this to ManageWiki be a better idea if we are actually moving this to LS.php?

At the moment, no I need to add a new type to managewiki first that does not allow current users to be entered as if a real user is entered to this field it woud make the account a system account

**Comment by Universal-Omega** [comment]
No changes made anymore?

**Comment by R4356th** [comment]
> No changes made anymore?

Sorry about that. I will make a new pull request.

**Comment by Universal-Omega** [comment]
> > No changes made anymore?
> 
> Sorry about that. I will make a new pull request.

No problem, and thanks!

**Pull request closed without merging**